vars/updatePropertyVersion.groovy

```diff
@@ -33,21 +33,15 @@ def call(body) {
       }
     }
     // only make a pull request if we've updated a version
-    def rs
     if (updated) {
-
       container(name: 'clients') {
 
         sh 'chmod 600 /root/.ssh-git/ssh-key'
         sh 'chmod 600 /root/.ssh-git/ssh-key.pub'
         sh 'chmod 700 /root/.ssh-git'
 
         sh "git push origin versionUpdate${uid}"
-
-        rs = flow.createPullRequest("[CD] Update release dependencies","${config.project}","versionUpdate${uid}")
-
       }
-      return rs
+      return flow.createPullRequest("[CD] Update release dependencies","${config.project}","versionUpdate${uid}")
```

Comment:
should you be inside the `container('clients')` to be able to make a `PR`?

Reply:
https://github.com/fabric8io-images/builder-clients (needs to be documented in the wiki :/) is just another container with a whole bunch of our binaries thrown in together. `createPullRequest` works by sending a HTTP post to github, so that should work anywhere as far as I understand now.